src/main.js

```diff
@@ -0,0 +1,7 @@
+import htmlParser from './htmlParser';
+import postscribe from './postscribe';
+
```

Comment:
Might I recommend adding `export default postscribe;`? It'll allow people to just `import postscribe from 'postscribe';` instead of having to do a named import (in ES6).